renderdoc/driver/d3d11/d3d11_manager.h

```diff
@@ -289,7 +289,10 @@ struct D3D11ResourceManagerConfiguration
 class D3D11ResourceManager : public ResourceManager<D3D11ResourceManagerConfiguration>
 {
 public:
-  D3D11ResourceManager(WrappedID3D11Device *dev) : m_Device(dev) {}
+  D3D11ResourceManager(WrappedID3D11Device *dev)
+      : m_State(CaptureState::BackgroundCapturing), ResourceManager(m_State), m_Device(dev)
```

Comment:
@baldurk, since D3* managers don't use the state, I'm setting it here to `CaptureState::BackgroundCapturing` to propagate the reference to the base manager. Let me know if you prefer another value here.

Reply:
We should be consistent and use the same scheme for all of the API managers even if they don't use the state right now.

As described in the other comment, if they all accept `CaptureState &driverState`, pass that through to `ResourceManager`, and remove any current `m_State` in any API manager they can share the parent's state.

Reply:
Yeah, sounds good, thanks.